plugin/Gui/AdminPageApiSettings.php

```diff
@@ -166,18 +166,51 @@ private function addFormModelGoogleMapsKey()
 
 	private function addFormModelGoogleBotSettings()
 	{
+		$pAdminViewController = new AdminViewController();
+		$labelGoogleBotIndexPdfExpose = __('Index PDF Brochure', 'onoffice-for-wp-websites');
 		$labelGoogleBotIndexPdfExpose = __('Allow indexing of PDF brochures', 'onoffice-for-wp-websites');
 		$pInputModeGoogleBotIndexPdfExpose = new InputModelOption('onoffice-settings', 'google-bot-index-pdf-expose',
 			$labelGoogleBotIndexPdfExpose, InputModelOption::SETTING_TYPE_BOOLEAN);
 		$pInputModeGoogleBotIndexPdfExpose->setHtmlType(InputModelOption::HTML_TYPE_CHECKBOX);
 		$pInputModeGoogleBotIndexPdfExpose->setValuesAvailable(1);
 		$pInputModeGoogleBotIndexPdfExpose->setValue(get_option($pInputModeGoogleBotIndexPdfExpose->getIdentifier()) == 1);
 		$pInputModeGoogleBotIndexPdfExpose->setDescriptionTextHTML(__('If you allow indexing, your search engine ranking can be negatively affected and your brochures can be available from search engines even months after the corresponding estate is deleted.','onoffice-for-wp-websites'));
+
+		$labelTitleAndDescription = __('Title and description', 'onoffice-for-wp-websites');
+		$pInputModeTitleAndDescription = new InputModelOption('onoffice-settings', 'title-and-description',
+			$labelTitleAndDescription, InputModelOption::SETTING_TYPE_NUMBER);
+		$pInputModeTitleAndDescription->setHtmlType(InputModelOption::HTML_TYPE_RADIO);
+		$pInputModeTitleAndDescription->setValuesAvailable([
+			__('Fill out', 'onoffice-for-wp-websites'),
+			__('Do not modify', 'onoffice-for-wp-websites'),
+		]);
+		$pInputModeTitleAndDescription->setValue(get_option($pInputModeTitleAndDescription->getIdentifier()));
+		$pInputModeTitleAndDescription->setDescriptionRadioTextHtmls([
```

Comment:
We will regret passing HTML into the setter. How about moving the formatting into the renderer? Would that work?